pkg/kube/kubehttp.go

```diff
@@ -0,0 +1,180 @@
+package kube
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	helmv2 "github.com/fluxcd/helm-controller/api/v2beta1"
+	kustomizev1 "github.com/fluxcd/kustomize-controller/api/v1beta1"
+	sourcev1 "github.com/fluxcd/source-controller/api/v1beta1"
+	"github.com/pkg/errors"
+	wego "github.com/weaveworks/weave-gitops/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	apiruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func CreateScheme() *apiruntime.Scheme {
+	scheme := apiruntime.NewScheme()
+	_ = sourcev1.AddToScheme(scheme)
+	_ = kustomizev1.AddToScheme(scheme)
+	_ = helmv2.AddToScheme(scheme)
+	_ = wego.AddToScheme(scheme)
+	_ = corev1.AddToScheme(scheme)
+
+	return scheme
+}
+
+var WeGONamespace = "wego-system"
+var WeGOCRDName = "apps.wego.weave.works"
+var FluxNamespace = "flux-system"
```

Comment:
Could we change this vars to constants and use them in other placers like this https://github.com/weaveworks/weave-gitops/pull/409/files#diff-49dcd96d19df2344f270f32a5b138783f8e8c3ddf2063ec83f998cbf2a29121eL126 ?

Reply:
>  change this vars to constants
We can, but changing to `const` just prevents runtime re-assignment. If someone added a change to re-assign, I would say "don't do that".

I guess `const` signals intent better in this case.

> use them in other placers like this

Good catch.

Reply:
cool